### ISSUE TITLE
[PATCH] Fix rendering of this code snippet in 'perldoc -f chomp'

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#14602]
Migrated from [rt.perl.org#124114](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=124114) (status was 'resolved')

Searchable as RT124114$

**Comment by p5pRT** [comment]
### From @karenetheridge

In text\, formerly rendered as&#8203;:  \("$/ = """\)
Now rendered as&#8203;: \("$/ = ''"\) which is a tad less confusing

**Comment by p5pRT** [comment]
### From @karenetheridge

<details><summary>0001-Fix-rendering-of-this-code-snippet-in-perldoc-f-chom.patch</summary>

```diff
From cfd500626b6f2321ce1c8c0fa72c1a7e0011ec8f Mon Sep 17 00:00:00 2001
From: Karen Etheridge <ether@cpan.org>
Date: Wed, 18 Mar 2015 19:13:56 -0700
Subject: [PATCH] Fix rendering of this code snippet in 'perldoc -f chomp'
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary="------------2.3.2"

This is a multi-part message in MIME format.
--------------2.3.2
Content-Type: text/plain; charset=UTF-8; format=fixed
Content-Transfer-Encoding: 8bit


In text, formerly rendered as:  ("$/ = """)
Now rendered as: ("$/ = ''") which is a tad less confusing
---
 pod/perlfunc.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)


--------------2.3.2
Content-Type: text/x-patch; name="0001-Fix-rendering-of-this-code-snippet-in-perldoc-f-chom.patch"
Content-Transfer-Encoding: 8bit
Content-Disposition: attachment; filename="0001-Fix-rendering-of-this-code-snippet-in-perldoc-f-chom.patch"

diff --git a/pod/perlfunc.pod b/pod/perlfunc.pod
index 9dc4cc6..c46f8c5 100644
--- a/pod/perlfunc.pod
+++ b/pod/perlfunc.pod
@@ -884,7 +884,7 @@ $INPUT_RECORD_SEPARATOR in the C<English> module).  It returns the total
 number of characters removed from all its arguments.  It's often used to
 remove the newline from the end of an input record when you're worried
 that the final record may be missing its newline.  When in paragraph
-mode (C<$/ = "">), it removes all trailing newlines from the string.
+mode (C<$/ = ''>), it removes all trailing newlines from the string.
 When in slurp mode (C<$/ = undef>) or fixed-length record mode (C<$/> is
 a reference to an integer or the like; see L<perlvar>) chomp() won't
 remove anything.

--------------2.3.2--



```
</details>

**Comment by p5pRT** [comment]
### From @jkeenan

On Wed Mar 18 19&#8203;:30&#8203;:33 2015\, ether wrote&#8203;:
> In text\, formerly rendered as&#8203;:  \("$/ = """\)
> Now rendered as&#8203;: \("$/ = ''"\) which is a tad less confusing

\+1

\-\- 
James E Keenan \(jkeenan@&#8203;cpan\.org\)

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From @khwilliamson

Thanks\, applied as cd3ae2edad0b39687eaf15efbfbf9cbc76a091a7
\-\- 
Karl Williamson

**Comment by p5pRT** [comment]
@khwilliamson - Status changed from 'open' to 'pending release'

**Comment by p5pRT** [comment]
### From @khwilliamson

Thank you for submitting this ticket\.

The issue should now be resolved with the release today of Perl v5\.22\, which is available at http&#8203;://www\.perl\.org/get\.html
\-\- 
Karl Williamson for the Perl 5 team

**Comment by p5pRT** [comment]
@khwilliamson - Status changed from 'pending release' to 'resolved'
